ci.sh

```diff
@@ -4,12 +4,11 @@ set -eo pipefail
 
 rm -rf ${HOME}/.chart-mate
 
-export CI=true
-
```

Comment:
There's no more usage of this variable since I split out bumpver into its own command.